File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Height (1-8): ");
    }
    while (h < 1 || h > 8);

    for (int i = 0; i < h; i++)
    {
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: WEEK 6/mario.py
# left pyramid for CS50

while True:
    try:
        h = int(input("Height: "))
        if h >= 1 and h <= 8:
            break
    except:
        pass

for i in range(1, h + 1):
    spaces = h - i
    blocks = i
    print(" " * spaces + "#" * blocks)
